src/server/pfs/cmds/cmds.go

```diff
@@ -310,6 +310,39 @@ $ pachctl list-commit foo master --from XXX
 	listCommit.Flags().IntVarP(&number, "number", "n", 0, "list only this many commits; if set to zero, list all commits")
 	rawFlag(listCommit)
 
+	printCommitIter := func(commitIter client.CommitInfoIterator) error {
+		if raw {
+			for {
+				commitInfo, err := commitIter.Next()
+				if err == io.EOF {
```

Comment:
I don't think it returns `EOF`.  This is just a helper function that's used by both `SubscribeCommit` and `FlushCommit`, and `FlushCommit` does return EOF I think.